student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(20, 'John')
```

Comment:
- [ ] The student class extends from the person class so it should also require the arguments used in creating a person.

  ```
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  ```